tests/Container/ContainerTest.php

```diff
@@ -795,3 +911,13 @@ function containerTestInject(ContainerConcreteStub $stub, $default = 'taylor')
 {
     return func_get_args();
 }
+
+class ContainerTestContextInjectInstantiations implements IContainerContractStub
+{
+    public static $instantiations = 0;
```

Comment:
It does not work. It's a left over from before I set it in the test itself. Thanks for catching this.